sections/introduction.include

```diff
@@ -627,23 +633,23 @@
 
   In the following example, the page has been made yellow-on-blue using CSS.
 
-  <xmp highlight="html">
-    <!DOCTYPE html>
-    <html>
-      <head>
-        <title>Sample styled page</title>
-        <style>
-          body {
-            background: blue;
-            color: yellow;
-          }
-        </style>
-      </head>
-      <body>
-        <h1>Sample styled page</h1>
-        <p>This page is just a demo.</p>
-      </body>
-    </html>
+  <xmp id="intro-early-example" highlight="html">
+  <!DOCTYPE html>
+  <html>
+    <head>
+      <title>Document title</title>
+      <style>
+        body {
+          background: blue;
+          color: yellow;
+        }
+      </style>
+    </head>
+    <body>
+      <h1>Styled page</h1>
+      <p>This is document has yellow text and a blue background.</p>
```

Comment:
"This is document" should read "This document"

Reply:
Hi @scottaohara, I've just fixed that typo and pushed an update